RPerformanceTracking/Private/_RPTClassManipulator+WKWebView.m

```diff
@@ -232,5 +232,35 @@ + (void)_swizzleWKWebViewNavDelegate:(id<WKNavigationDelegate>)delegate
                   onClass:recipient
         newImplementation:imp_implementationWithBlock(didReceiveServerRedirect_swizzle_blockImp)
                     types:"v@:@@"];
+
+    // MARK: WKNavigationDelegate webView:decidePolicyForNavigationResponse:decisionHandler:
+    id decidePolicyForNavigationResponse_swizzle_blockImp = ^(id<NSObject> selfRef, WKWebView *webView, WKNavigationResponse *navigationResponse, void (^decisionHandler)(WKNavigationResponsePolicy)) {
+        RPTLogVerbose(@"decidePolicyForNavigationResponse_swizzle_blockImp called");
+
+        SEL selector = @selector(webView:decidePolicyForNavigationResponse:decisionHandler:);
+        IMP originalImp = [_RPTClassManipulator implementationForOriginalSelector:selector class:selfRef.class];
+        if (originalImp)
+        {
+            ((void(*)(id, SEL, id, id, id))originalImp)(selfRef, selector, webView, navigationResponse, decisionHandler);
+        }
+        else
+        {
+            decisionHandler(WKNavigationResponsePolicyAllow);
```

Comment:
Although the handler _should_ be non NULL perhaps we should do a NULL check before calling it.

Reply:
The decisionHandler block comes from WebKit so should always be valid. Therefore an additional check is unnecessary.